.github/workflows/push-translation.yml

```
@@ -0,0 +1,55 @@
+name: Push translation sources
+
+on:
+  pull_request:
+    paths:
+      - '*.rst'
+jobs:
+  push-translation-sources:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set branch name from source branch
+        run: echo "BRANCH_NAME=${GITHUB_HEAD_REF##*/}" >> $GITHUB_ENV
+
+      - name: Start translation service deployment
+        uses: bobheadxi/deployments@v0.5.2
+        id: translation
+        with:
+          step: start
+          token: ${{secrets.GITHUB_TOKEN}}
+          env: ${{env.BRANCH_NAME}}
```

why not translation-branchname?

```suggestion
          env: translation-${{env.BRANCH_NAME}}
```